core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java

```diff
@@ -172,20 +174,31 @@ Map<String,String> modifyProperties(Consumer<Map<String,String>> mapMutator)
    * @return a list of locations in <code>hostname:port</code> form.
    * @since 2.1.0
    */
+  @Deprecated
   List<String> getManagerLocations();
 
   /**
    * Returns the locations of the active scan servers
    *
    * @return A set of currently active scan servers.
    */
+  @Deprecated
   Set<String> getScanServers();
 
+  /**
+   * Returns the locations of the active scan servers
+   *
+   * @return A set of currently active scan servers.
+   * @since 3.0.0
+   */
+  Set<Server> getServers(ServerType type);
```

Comment:
Wondering how this method could also return information about a servers group.  Was thinking through the following possible options.

 1.  Group info could be added to the Sever class.
 2. This method could return something like Map<Server, ServerInfo> where ServerInfo includes the group
 3. Could add a new method to `String getGroup(Server server)` to InstanceOperations.

Reply:
I added `resourceGroup` to `Server` in ea67851. This is a little before it's time, as this is fully implemented in the elasticity branch.

Reply:
> This is a little before it's time, as this is fully implemented in the elasticity branch.

What is the diff between main and elasticity that impacts this PR?

Reply:
My comment is that the name `resourceGroup` is a little before it's time. In 2.1 ScanServers have groups and Compactors have queues, which is what will be returned here for 3.0. In elasticity as you know, we have centralized on the name `group` in the properties and code, and I have been calling the feature `resource groups` as `groups` is too vague and the way that they are intended to be used is to create groups of resources dedicated to running tservers/sservers/compactors for specific use-cases.

Reply:
May make sense to defer adding the resourceGroup to the server type to the elasticity branch.  Could still add these new types in 3.0 and not have the resource group method in 3.0.  Thinking if we release something in 3.x that still may still change in the elasticity branch that it could cause problems if it does change.

Reply:
@ctubbsii  said he wanted to review this too. I'll wait for his feedback and make this changes along with anything that he suggests.

Reply:
Removed resourceGroup, can re-add this in elasticity branch